cmd/api/src/api/v2/assetgrouptags.go

```diff
@@ -158,3 +158,28 @@ func (s *Resources) GetAssetGroupTag(response http.ResponseWriter, request *http
 		api.WriteBasicResponse(request.Context(), getAssetGroupTagResponse{Tag: assetGroupTag}, http.StatusOK, response)
 	}
 }
+
+type getAssetGroupTagMemberCountsResponse struct {
+	TotalCount int            `json:"total_count"`
+	Counts     map[string]int `json:"counts"`
+}
+
+func (s *Resources) GetAssetGroupTagMemberCountsByKind(response http.ResponseWriter, request *http.Request) {
+	if tagId, err := strconv.ParseInt(mux.Vars(request)[api.URIPathVariableAssetGroupTagID], 10, 32); err != nil {
```

Comment:
This seems easier? No additional parameters, and it returns an int so you don't have to cast it in the next line. Also TIL that `Atoi` contains optimizations for common cases that make it faster than `ParseInt`.
```suggestion
	if tagId, err := strconv.Atoi(mux.Vars(request)[api.URIPathVariableAssetGroupTagID]); err != nil {
```